db/models.py

```diff
@@ -14,3 +14,35 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(to=Actor, related_name="actors")
+    genres = models.ManyToManyField(to=Genre, related_name="genres")
```

Comment:
The `related_name` for the `genres` field should be more descriptive and not the same as the field name. Consider using a name like `movies_in_genre` to avoid confusion and potential conflicts.